html-forms/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <div>
+      <label>Email:<br/>
+        <input type="text" name="email"/></label><br/>
+      Why is coding cool?<br/>
```

Comment:
Instead of using `br` elements to make space, you should be putting elements that need to be on their own line in a `div` element